__tests__/components/FriendRequestSidebarOptions/helpers.test.ts

```diff
@@ -83,21 +130,51 @@ describe('PusherClientHandler handleRequest tests', () => {
 
     test('the existing count is 1, so 2 is passed to the setter',()=>{
         client = new PusherClientHandler('stub_id', 1)
-        const func = client.handleRequest(setterSpy)
+        const func = client.incrementCount(setterSpy)
         func()
         expect(setterSpy).toHaveBeenCalledWith(2)
     })
 
     test('the existing count is 8, so 9 is passed to the setter',()=>{
         client = new PusherClientHandler('stub_id', 8)
-        const func = client.handleRequest(setterSpy)
+        const func = client.incrementCount(setterSpy)
         func()
         expect(setterSpy).toHaveBeenCalledWith(9)
     })
 })
 
+describe('PusherClientHandler decrementCount tests', () => {
+    let client: PusherClientHandler;
+    let setterSpy: jest.Mock
+    beforeEach(()=>{
+        setterSpy = jest.fn()
+    })
+
+    test('the existing count is 1, so 0 is passed to the setter',()=>{
+        client = new PusherClientHandler('stub_id', 1)
+        const func = client.decrementCount(setterSpy)
+        func()
+        expect(setterSpy).toHaveBeenCalledWith(0)
+    })
+
+    test('the existing count is 0, dont call the setter',()=>{
```

Comment:
uncertain if this will cause issues, but the functionality isn't central, so it's okay to see what happens then fix it down the line.